renderdoc/driver/vulkan/wrappers/vk_get_funcs.cpp

```diff
@@ -1167,3 +1167,28 @@ VkResult WrappedVulkan::vkGetDeferredOperationResultKHR(VkDevice device,
 {
   return ObjDisp(device)->GetDeferredOperationResultKHR(Unwrap(device), operation);
 }
+
+void WrappedVulkan::vkGetAccelerationStructureBuildSizesKHR(
+    VkDevice device, VkAccelerationStructureBuildTypeKHR buildType,
+    const VkAccelerationStructureBuildGeometryInfoKHR *pBuildInfo,
+    const uint32_t *pMaxPrimitiveCounts, VkAccelerationStructureBuildSizesInfoKHR *pSizeInfo)
+{
+  ObjDisp(device)->GetAccelerationStructureBuildSizesKHR(Unwrap(device), buildType, pBuildInfo,
+                                                         pMaxPrimitiveCounts, pSizeInfo);
+}
+
+VkDeviceAddress WrappedVulkan::vkGetAccelerationStructureDeviceAddressKHR(
+    VkDevice device, const VkAccelerationStructureDeviceAddressInfoKHR *pInfo)
+{
+  VkAccelerationStructureDeviceAddressInfoKHR info = *pInfo;
+  info.accelerationStructure = Unwrap(info.accelerationStructure);
+  return ObjDisp(device)->GetAccelerationStructureDeviceAddressKHR(Unwrap(device), &info);
+}
+
+void WrappedVulkan::vkGetDeviceAccelerationStructureCompatibilityKHR(
+    VkDevice device, const VkAccelerationStructureVersionInfoKHR *pVersionInfo,
+    VkAccelerationStructureCompatibilityKHR *pCompatibility)
+{
+  ObjDisp(device)->GetDeviceAccelerationStructureCompatibilityKHR(Unwrap(device), pVersionInfo,
+                                                                  pCompatibility);
```

Comment:
We should forcibly fail this query rather than passing it down to the API, to ensure the user doesn't try to deserialise an acceleration structure.